modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptClientCodegen.java

```diff
@@ -142,6 +145,14 @@ public TypeScriptClientCodegen() {
         }
         frameworkOption.defaultValue(FRAMEWORKS[0]);
 
+        cliOptions.add(new CliOption(TypeScriptClientCodegen.PLATFORM_SWITCH, TypeScriptClientCodegen.PLATFORM_SWITCH_DESC));
```

Comment:
@bodograumann Is it possible that there's a bug here? First a new `CliOption` object is created and added to `cliOptions`, then another `CliOption` is created, which isn't added to `cliOptions`.

Reply:
I think it is fine. The framework option is added to cliOptions further down:
```cliOptions.add(frameworkOption);```

Reply:
But `platformOption`/`PLATFORM_SWITCH` (see l.148, l. 149-154) isn't or am i blind?

Reply:
Ah now I see it. Yes I guess there is some redundant code.
Creating the duplicate CliOption object doesn't seem to cause any problems ^^.
Still, good catch, I should fix that.